frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditEmail/EditEmail.tsx

```diff
@@ -130,10 +131,18 @@ export const EditEmail = ({ field }: EditEmailProps): JSX.Element => {
 
   const { data: form } = useCreateTabForm()
   const isPdfResponseEnabled = useMemo(
-    () => form?.responseMode !== FormResponseMode.Multirespondent,
-    [form],
+    () =>
```

Comment:
We might not need to use `useMemo` here. since it is just a boolean check and `useMemo` introduces more overhead